BaseFilter/sections/antiadblock.txt

```diff
@@ -1088,6 +1088,8 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 ! NOTE: Regular rules
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/180788
+telegratishd.com##.site-access-popup

```

Comment:
There is also an iframe `cablevisionhd.com` with the same anti-adblock popup.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/32f96b56-1d9a-4598-abd6-9115629b0f6b)

</details>

```suggestion
cablevisionhd.com,telegratishd.com##.site-access-popup
```